config/mail.php

```diff
@@ -57,7 +57,7 @@
 
         'sendmail' => [
             'transport' => 'sendmail',
-            'path' => '/usr/sbin/sendmail -bs',
+            'path' => env('SENDMAIL','/usr/sbin/sendmail -bs'),
```

Comment:
fixing with the proper variable name: https://github.com/laravel/laravel/blob/21ad6d6915df7d8b3825e6d7a7f47f716600e87b/config/mail.php#L70C28-L70C46
```suggestion
            'path' => env('MAIL_SENDMAIL_PATH', '/usr/sbin/sendmail -bs -i'),
```